src/axom/core/Path.cpp

```diff
@@ -74,6 +74,12 @@ std::string Path::baseName() const
 
 std::string Path::dirName() const { return static_cast<std::string>(parent()); }
 
+std::pair<std::string, std::string> Path::split() const
+{
+  Path result(*this);
+  return std::make_pair(result.dirName(), result.baseName());
```

Comment:
Suggestion: Since both functions are const and return copies, I think we can do this without the initial `result` variable.

E.g.
```suggestion
  return std::make_pair(dirName(), baseName());
```